### PR TITLE
Switch to buildkite

### PR DESCRIPTION
**Pull request by balthazar**

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+steps:
+  - name: ":docker:"
+    plugins:
+      docker-compose#v1.5.2:
+        build: fusion-core
```
Comment (KevinGrandon) [inline .buildkite/pipeline.yml]: This should be `build: luma-gl`

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - wait
+
+  - name: ":eslint:"
+    command: "yarn lint"
+    plugins:
+      docker-compose#v1.5.2:
+        run: luma-gl
+
+  - name: ":hourglass:"
+    command: "sh /etc/init.d/xvfb && yarn test"
```
Comment (KevinGrandon) [inline .buildkite/pipeline.yml]: Consider making this a file so it's easier to reproduce locally. E.g., I want to be able to run the same command with something like: 

```
docker-compose run luma-gl .buildkite/test.sh
```

File: test/bench/node.js [hunk 3]
```diff
@@ -21,6 +21,9 @@
 // Enables ES2015 import/export in Node.js
 require('reify');
 
+/* global global */
```
Comment (KevinGrandon) [inline test/bench/node.js]: Is this change needed as part of this?

**Comment by 1chandu** [comment]
>This is great, nice work! Need to finish the process to get Buildkite fully enabled and we should be able to land this soon. (We could potentially land this earlier if we want to leave the .travis.yml file)

@balthazar , I am not an expert of this area, but as long as it doesn't block landing PRs I am fine landing it before Buildkite process is finished.

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (balthazar) [reply .buildkite/pipeline.yml]: ahhh failed again

File: test/bench/node.js (hunk 3 above)
Comment (balthazar) [reply test/bench/node.js]: yep

**Comment by balthazar** [comment]
I would say my biggest concern is that people have to signup to buildkite in order to see the build status, which should be something public. I guess we could bring this up to them if they are open to the idea.

**Comment by KevinGrandon** [comment]
> biggest concern is that people have to signup

This is actually something they're working on, and we should be able to have open build pages soon.

**Comment by balthazar** [comment]
That's good to know, let's see next week how it goes about the security stuff, if it doesn't look like it will happen next week I'll add back the `travis.yml` and merge
